taxi/views.py

```diff
@@ -44,6 +62,27 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+
+    def get_success_url(self):
+        return reverse("taxi:car-detail", kwargs={"pk": self.object.id})
+
+
+class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Car
+    fields = "__all__"
+
+    def get_success_url(self):
+        return reverse("taxi:car-detail", kwargs={"pk": self.kwargs["pk"]})
```

Comment:
it is better to use a method **or** attr for success url in every view, to make code more consistant